base/server/src/main/java/com/netscape/cmscore/dbs/Repository.java

```diff
@@ -622,9 +638,11 @@ public boolean hasRangeConflict() throws EBaseException {
 
             logger.info("Repository: Searching for conflicting entries");
 
+            String minSerial = idGenerator == IDGenerator.LEGACY_2 ? 
+                    mMinSerialNo.toString() : mMinSerialNo.toString(mRadix);
```

Comment:
Could you add a note here saying that `beginRange` is always decimal and the code that generates a hexadecimal value for SSNv1 is incorrect?